### DISCUSSION TITLE
Welcome to LazyVim Discussions!

### DISCUSSION THREAD

**Discussion by folke** [LazyVim/LazyVim#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nxy7** [comment]
Hello, just wanted to say that ur job with lazy was amazing and now you created distro around it. I was kinda leaning into using helix because I don't like configuring configs too much but you made me stay with neovim. Great work as always ;-)

**Reply by folke** [reply]
Thanks! Any feedback is welcome before I officially launch LazyVim.

**Comment by 1Mike1** [comment]
After installation whenever i am trying to save the file it always delete the very first line, similarly when i opening project it is loading the workspace and while doing this it is again deleting the first line.

I tried reinstalling, also did the clean setup but no luck. 

Please refer the attach video for the reference.
[nvim_issue.webm](https://github.com/LazyVim/LazyVim/assets/45176462/8a72f18d-703a-4de7-9fde-2887ac75d9c2)

**Reply by folke** [reply]
That's caused by a nasty upstream Neovim bug. Upgrade to the latest nightly. That should solve most of it. Just Telescope is still messed up

**Comment by msMatix** [comment]
Hey !
Really cool setup. I've been using nvim for almost 2 years now and started with a different framework.  When I switched, this was much easier for me and you can see that everything has been thought through. Thanks a lot. With this I can work great !

**Comment by ZhangL3** [comment]
Hello!
LazyVim is amazing. Thanks for your job!
I have a quastion, how to disable the leap plugin. I want to us 's' as REPLACE as normal Vim.
Ich have add 
`
return {
  { "flazyvim.plugins.extras.editor.leap, enabled = false },
}
`
to disabled.lua
But there es error: no such plugin.

**Reply by dpetka2001** [reply]
`leap` is not enabled by default on Lazyvim. `flash` is its replacement. Try to disable that.

**Reply by ZhangL3** [reply]
It works with
`
return { { "folke/flash.nvim, enabled = false }, }
`
Thank you very much!

**Comment by timjackleus** [comment]
Hello! Very thankful for this amazing distro!

I've been using vim/neovim for the past 4 years or so. I loved personalizing my config, but with the introduction of native lsp, treesitter etc (which have been making neovim itself so much better!), things would often break, especially when I was pressed for time. Since switching to LazyVim, everything feels more stable. I really appreciate the fundamental tools that now work seamlessly – like lsp, treesitter, prettier, eslint, and the rest. Thanks again!

**Comment by rasivasu** [comment]
Hello @folke,

Thank you for creating LazyVim.  I have been using LazyVim for six months and I am loving it.  

Last week I tried Helix and felt that the popup menu at the bottom was instantaneous.  When I cam back to using Neovim with LazyVim, I started noticing that I had to wait for a fraction of a second for the menu to come up.  Is there any effort either from your end or from the Neovim community to improve the response time?  Or can I turn off a few things in LazyVim to make the response snappier?

**Reply by dpetka2001** [reply]
Check out `:h 'timeoutlen'`.

**Reply by folke** [reply]
De delay is on purpose.

**Comment by rasivasu** [comment]
That is awesome.  The response is a lot snappier at 50 ms.  Thank you very
much :-)

On Sun, Dec 3, 2023 at 3:37 PM Folke Lemaitre ***@***.***>
wrote:

> De delay is on purpose.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LazyVim/LazyVim/discussions/1#discussioncomment-7742928>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIZULTNWGLVDFNJRADOFJ3YHRFPRAVCNFSM6AAAAAATN7TVW2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TONBSHEZDQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Regards,
Ravi

**Comment by gmhpfc** [comment]
I got error while using LazyVim config loading with Lazy.nvim, iam not sure is it is a virus isuue or somthing else, the pakage got installed after deleting and reinstalling it, I am using ubuntu 23.10 on Macbook Pro 
 
LazyVim 26.95ms  start **status failed**

        You have local changes in `/home/vp/.local/share/nvim/lazy/LazyVim`:
                  * README-DE.md
                Please remove them to update.
                You can also press `x` to remove the plugin and then `I` to install it again.

**Comment by heyyshaarib** [comment]
![Screenshot 2024-11-10 082407](https://github.com/user-attachments/assets/7c36da3f-92a2-4091-bf8d-784fabbf371f)
can anyone help me with this?

**Comment by Lucy2171** [comment]
How can I connect it with Facebook and WhatsApp
